shared/actions/teams.tsx

```diff
@@ -1143,6 +1143,43 @@ const teamChangedByName = (
   return getLoadCalls()
 }
 
+const teamRoleMapChangedUpdateLatestKnownVersion = (
+  _: TypedState,
+  action: EngineGen.Keybase1NotifyTeamTeamRoleMapChangedPayload
+) => {
+  const {newVersion} = action.payload.params
+  return TeamsGen.createSetTeamRoleMapLatestKnownVersion({version: newVersion})
+}
+
+const refreshTeamRoleMap = async (logger: Saga.SagaLogger) => {
```

Comment:
Instead of calling this from two sagas, you can have this be the saga and listen to both actions directly.
```
const refreshTeamRoleMap = 
  async (
    state: TypedState, 
    action: EngineGen.Keybase1NotifyTeamTeamRoleMapChangedPayload | ConfigGen.BootstrapStatusLoadedPayload,
    logger: Saga.SagaLogger
  ) =>
```
Then within if `action.type` is the notification, bail loading if the version number hasn't changed.